oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,249 @@
+welcoming to all in the community, and have shown the ability to deliver high quality work over time.
+
+Core Contributor Roles
+----------------------
+
+There are many contributors to the Open edX platform that participate in a variety of ways - writing
+code for the Open edX software, translating the project, performing Quality Assurance tasks, participating
+in the Marketing working group, writing documentation, and more. At the time of writing we have identified
+the following roles: Project Manager, Translation Lead, Code Contributors, Forum Moderators, UI/UX Designers,
+Marketing Team Members, Documentation Writers, QA, and Product Managers.
+
+
+All CCs are granted some Rights to the project as part of participating - the ability to
+merge code, the final say on marketing copy, etc. In exchange for these Rights to contribute, we ask that
+CCs abide by their Responsibilities. For more on the CC roles, and Rights & Responsibilities, see:
+
+- `Core Contributor Role Definitions <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2759460357/Core+Contributor+Role+Definitions>`_
+- `Core Contributor Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/2952003698/Core+Contributor+Rights+Responsibilities+excluding+code+contributors>`_
+- `Code Committer Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Rights+Responsibilities+for+Code+Contributors>`_
+
+Note that roles vary in what they need from members; the wiki will be the most up to date source
+of what each role entails and what types of commitments are expected.
+
+.. _program administration:
+
+Core Contributor Program Administration
+---------------------------------------
+
+The Core Contributor Program is administered by The Center for Re-imagining
+Learning (tCRIL). One or more persons at tCRIL (the "Program administrator(s)")
+work with the community to make sure the program runs smoothly: this entails
+facilitating communication amongst community members, `selecting new members`_,
+and ensuring that existing members renew their commitment to the community and
+abide by the Code of Conduct. Program administrators are empowered to monitor
+Core Contributor conduct, particularly whether they follow the "3 Cs"
+(Commitment, Conduct, Caliber), and may issue reminders, suspensions, and even
+removal from the Program. Program administrators can be contacted at
+``cc-program-admins@tcril.org``.
+
+The tCRIL administrator(s) are the point people for making sure that needed
+agreements are signed - this includes a Program Agreement between the CC and
+tCRIL (sent to you by a Program Administrator); for code contributors, an
+additional CLA (signed when you make your first PR); and for new entities
+joining the community, an Entity Contributor Agreement (please communicate with
+a Program Administrator about this).
+
+We are additionally asking that all employers of CCs sign the `Revised
```

Comment:
Mentioning employers here precludes individuals becoming CCs independent of their day job.  Are we OK with that?

Reply:
No, I'll clarify this point.

Reply:
@nedbat How would you recommend rewording this? The point is, if you employ a CC we hope you will sign the agreement. I'm not sure I agree that this wording precludes individuals.

Reply:
For example, I could be employed by a company that has nothing to do with Open edX. I wouldn't expect that employer to sign a commitment. I'd be committing my own time as an individual.

Reply:
Ah! I see how that could be read that way! Hmm... what about,

`We are additionally asking that all companies and teams joining the Open edX community sign ...`?

Reply:
Ah! I see how that could be read that way! Hmm... what about,

`We are additionally asking that all companies and teams joining the Open edX community sign ...`?